pkg/scheduler/preemption/preemption.go

```diff
@@ -221,7 +219,7 @@ func (p *Preemptor) classicalPreemptions(preemptionCtx *preemptionCtx) []*Target
 		Requests:          preemptionCtx.workloadUsage.Quota,
 		WorkloadOrdering:  p.workloadOrdering,
 	}
-	candidatesGenerator := classical.NewCandidateIterator(hierarchicalReclaimCtx, p.enabledAfs, preemptionCtx.frsNeedPreemption, preemptionCtx.snapshot, p.clock, CandidatesOrdering)
+	candidatesGenerator := classical.NewCandidateIterator(hierarchicalReclaimCtx, p.enabledAfs, preemptionCtx.frsNeedPreemption, preemptionCtx.snapshot, p.clock, fairsharing.CandidatesOrdering)
```

Comment:
Can we move CandidatesOrdering to preemptioncommon? classical should not be using fairsharing package. Consider doing this in a prepatory PR (I'll leave this up to you).

Reply:
Yes, I will do that

Reply:
Done